source/hardware/controllers/hercules_djcontrol_jogvision.rst

```diff
@@ -0,0 +1,127 @@
+     - 1
+     - 0\|1
+     - (0=disabled; 1=use outer jog leg to indicate where to slide the pitch controller; see also 'CFG.fine.beatHelpSensitivity' variable)
+   * - beatActiveMode
+     - follow
+     - normal\|reverse\|blink\|follow\|fill\|bounce\|alternate\|off\|on
+     - defines how the beats leds will lite
+
+.. list-table:: *FINE TUNNING Variables* (you may modify them, but not too far from the default values...)
+   :widths: 25 25 25 50
+   :header-rows: 1
+
+   * - VARIABLE NAME
+     - DEFAULT VALUE
+     - AVAILABLE VALUES
+     - DESCRIPTION
+   * - beatHelpSensitivity
+     - 0.5
+     - \[0..1)
+     - Max distance (float) in BPMs to be considered as a match (in use if CFG.user.beatHelper=1): the lower, the more sensitive. Values equal or bigger than 1 are reset to 0.9 and a warning is printed
+   * - quickMoveFactor
+     - 0.002
+     - \[0..1\]
+     - the smaller (float), the slower MODE+JogWheel will move 'playposition' (when such channel is NOT playing)
+   * - quickBrowseFactor
+     - 10
+     - \[0..inf\]
+     - the bigger, the faster MODE+Browser jog will move the cursor position in the library up/down
+   * - spinBackBrakeFactor
+     - 100
+     - (0..5000\]
+     - the bigger, the softer the brake will be applied (0 = immediate stop; >=5000 = it will take almost forever to stop)
+   * - spinBackInitialSpeed
+     - 6
+     - (0..200\]
+     - the bigger, the stronger will be the "back" impulse (1 = no spinbak, but stop and start sloooowly)
+   * - mixGainFactor
+     - 0.1
+     - (0..1)
+     - (float) the bigger, the faster the Pregain or Mix level will be updated
+
+All controls *act as labeled*, but there are some **additional** functionalities that have been added:
+
+- MODE+Loop ON                  : set a loop_in mark (with curently defined loop_size), activate it, and enable slip mode  
+- MODE+Loop X 1/2 / X 2         : do a 'beatjump_size' beats beatjump backward/forward  
+- MODE+Loop Size Knob           : decrease/increase pitch (only key, not tempo!)  
+- MODE+JogWheel plate (playing) : scratch with 'Slip' ON (deactivate 'Slip' when plate is released)  
+- MODE+JogWheel plate (stopped) : move song position backward/forward faster by 'quickMoveFactor' factor  
+- MODE+Browser Knob Turn        : move library selected position in groups of 'quickBrowseFactor' elements forward/backward  
+- MODE+LOAD A|B                 : toggle 'quantize' for deck where MODE key is pressed  
+- SHIFT+LOAD A|B                : eject track from deck where SHIFT key is pressed  
+- SHIFT+Browser Knob Press      : activate (double-click) currently selected item in browser  
+- SHIFT+Loop Size Knob          : move existing loop forward/backward  
+- SHIFT+JogWheelTouch           : do a 'backspin' with 'spinBackBrakeFactor' and 'spinBackInitialSpeed' factors  
+- SHIFT+MultiFX                 : set beatgrid to current position  
+- SHIFT+Air control Filter      : do the reverse than standard, that is, high-pass filter  
```

Comment:
Please run `git pull https://github.com/mixxxdj/manual.git manual-2.3.x` and push to this branch, then apply this suggestion:

```suggestion
=========================================================  =============================================================
Control                                                    Description
=========================================================  =============================================================
:hwlabel:`MODE` + :hwlabel:`LOOP ON                        Set a loop_in mark (with curently defined loop_size), activate it, and enable slip mode
:hwlabel:`MODE` + :hwlabel:`LOOP X½` / :hwlabel:`LOOP X2`  Do a beatjump backward/forward
:hwlabel:`MODE` + :hwlabel:`LOOP SIZE` knob                Decrease/Increase pitch (only key, not tempo!)
:hwlabel:`MODE` + Jog Wheel plate (playing)                Scratch with 'Slip' on (deactivate 'Slip' when plate is released)
:hwlabel:`MODE` + Jog Wheel plate (stopped)                Move song position backward/forward faster
:hwlabel:`MODE` + :hwlabel:`BROWSER` knob turn             Move library selected position multiple elements forward/backward
:hwlabel:`MODE` + :hwlabel:`LOAD` buttons                  Toggle 'quantize' for deck where :hwlabel:`MODE` button is pressed
:hwlabel:`SHIFT` + :hwlabel:`LOAD` buttons                 Eject track from deck where :hwlabel:`SHIFT` key is pressed
:hwlabel:`SHIFT` + :hwlabel:`BROWSER` knob press           Activate (double-click) currently selected item in browser
:hwlabel:`SHIFT` + :hwlabel:`LOOP SIZE` knob               Move existing loop forward/backward
:hwlabel:`SHIFT` + Jog Wheel touch                         Do a backspin
:hwlabel:`SHIFT` + `:hwlabel:`MULTI FX`                    Set beatgrid to current position
:hwlabel:`SHIFT` + :hwlabel:`AIR CONTROL` Filter           Do the reverse than standard, that is, high-pass filter
=========================================================  =============================================================
```